src/app/src/test/kotlin/org/vaccineimpact/api/tests/logic/BurdenEstimateLogic/BaseBurdenEstimateLogicTests.kt

```diff
@@ -0,0 +1,86 @@
+package org.vaccineimpact.api.tests.logic.BurdenEstimateLogic
```

Comment:
The BurdenEstimatesLogicTests class was super long so I split it up further and introduced this base class for shared logic